sessions/html-forms/zoo/css/styles.css

```
@@ -0,0 +1,61 @@
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica, Arial, sans-serif,
+		'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+	margin: 20px auto;
+	max-width: 300px;
```

max-width is to low: I think 500px would be better:

```suggestion
	max-width: 500px;
```